### PR TITLE
feat: image, input

### PR DESCRIPTION
**Pull request by ddivyansh18**
- Added responsive image, square image, circular image
- Added input with valid input and invalid input styles

### CODE REVIEW

File: Components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/* * {
+  outline: 1px solid red;
+} */
+
```
Comment (romabulani) [inline Components/Image/image.css]: You can remove comments if not in use

File: Components/Image/image.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        <img
+          src="https://cdn0.iconfinder.com/data/icons/font-awesome-brands-vol-2/512/steam-square-256.png"
+          alt=""
+          class="square-image"
+        />
+        <img
+          src="https://cdn0.iconfinder.com/data/icons/font-awesome-brands-vol-2/512/youtube-square-256.png"
+          alt=""
+          class="square-image"
```
Comment (romabulani) [inline Components/Image/image.html]: Give some value to alt attribute

File: Components/Input/input.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/* * {
+  outline: 1px solid red;
+} */
```
Comment (romabulani) [inline Components/Input/input.css]: Here also you can remove the comments

File: Components/Image/image.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
+/* * {
+  outline: 1px solid red;
+} */
+
+.image-container {
+  display: flex;
+  height: 10rem;
+  width: 50rem;
+  flex-direction: row;
```
Comment (codewithrishu) [inline Components/Image/image.css]: flex direction row is by default, you don't have write that

**Comment by romabulani** [comment]
Apart from these 2 changes, code looks good to merge

- You can remove comments if not in use
- You should give value to alt attribute

File: Components/Image/image.html [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    <title>Image</title>
+    <link rel="stylesheet" href="../default.css" />
+    <link rel="stylesheet" href="image.css" />
+  </head>
+  <body>
+    <div class="column-container">
+      <h3>Responsive Image</h3>
+      <div class="image-container">
+        <img
+          src="https://images.unsplash.com/photo-1498036882173-b41c28a8ba34?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1964&q=80"
```
Comment (codewithrishu) [inline Components/Image/image.html:16]: write for alt also, don't leave it empty

File: Components/Image/image.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
+/* * {
+  outline: 1px solid red;
+} */
```
Comment (codewithrishu) [inline Components/Image/image.css]: remove the unwanted commented code

File: Components/default.css [hunk 7]
```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
 * {
   margin: 0;
   padding: 0;
```
Comment (codewithrishu) [inline Components/default.css:5]: other then that, to me everything looks good, 
just a suggestion- you can write comments above links inserted.

File: Components/Input/input.css (hunk 3 above)
Comment (ddivyansh18) [reply Components/Input/input.css]: Removed comments from both the files

File: Components/Image/image.html (hunk 2 above)
Comment (ddivyansh18) [reply Components/Image/image.html]: Done

File: Components/Image/image.css (hunk 1 above)
Comment (ddivyansh18) [reply Components/Image/image.css]: Removed comments from both the files

File: Components/Image/image.css (hunk 6 above)
Comment (ddivyansh18) [reply Components/Image/image.css]: Removed comments from both the files

File: Components/Image/image.html (hunk 5 above)
Comment (ddivyansh18) [reply Components/Image/image.html:16]: Done

File: Components/Image/image.css (hunk 4 above)
Comment (ddivyansh18) [reply Components/Image/image.css]: Removed that property from that class

**Comment by ddivyansh18** [comment]
@romabulani @codewithrishu  I have made the requested changes, let me know if there are any more suggestions. If not, please approve the PR, thanks.

**Review by lomesshh — APPROVED** [review]
looks good to merge

**Review by codewithrishu — APPROVED** [review]
looks good.

**Review by romabulani — APPROVED** [review]

**Comment by romabulani** [comment]
> @romabulani @codewithrishu I have made the requested changes, let me know if there are any more suggestions. If not, please approve the PR, thanks.

Approved

**Pull request merged**